akka-http-core/src/main/scala/akka/http/scaladsl/model/Multipart.scala

```diff
@@ -344,8 +344,18 @@ object Multipart {
      * To create an instance with several parts or for multiple files, use
      * `FormData(BodyPart.fromFile("field1", ...), BodyPart.fromFile("field2", ...)`
      */
+    @deprecated("Use `fromPath` instead", "2.4.5")
     def fromFile(name: String, contentType: ContentType, file: File, chunkSize: Int = -1): Multipart.FormData =
-      Multipart.FormData(Source.single(Multipart.FormData.BodyPart.fromFile(name, contentType, file, chunkSize)))
+      fromPath(name, contentType, file.toPath, chunkSize)
+
+    /**
+      * Creates a FormData instance that contains a single part backed by the given file.
+      *
+      * To create an instance with several parts or for multiple files, use
+      * `FormData(BodyPart.fromPath("field1", ...), BodyPart.fromPath("field2", ...)`
+      */
```

Comment:
Scaladoc formatting should be as is in the rest of the project (apply everywhere):

```scala
   /**
     * Creates a FormData instance that contains a single part backed by the given file.
     *
     * To create an instance with several parts or for multiple files, use
     * `FormData(BodyPart.fromPath("field1", ...), BodyPart.fromPath("field2", ...)`
     */
```